lineman/app/components/memberships_page/cancel_invitation_form/cancel_invitation_form.haml

```diff
@@ -0,0 +1,12 @@
+%form{ng-submit: 'submit()'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
+      %span.sr-only{translate: 'common.action.cancel'}
+    %h1.lmo-h1{translate: 'cancel_invitation_form.heading'}
+
+  .modal-body
+    %span{translate: 'cancel_invitation_form.question'}
+  .modal-footer
+    %button.btn.btn-danger{type: 'submit', translate: 'cancel_invitation_form.submit'}
```

Comment:
if you put a class on it.. and the class is not there.. then I think you're editing the wrong file/another file is being used and we need to get to the bottom of that.